augur/metrics/issue/issue.py

```diff
@@ -111,7 +111,7 @@ def issues_first_time_closed(self, repo_group_id, repo_id=None, period='day', be
                     AND repo.repo_id = issues.repo_id
                     AND issues.pull_request IS NULL 
                     AND issues.issue_id = issue_events.issue_id
-                    And issue_events.created_at BETWEEN :begin_date AND :end_date
+                    And issue_events.created_at BETWEEN to_timestamp(:begin_date, 'YYYY-MM-DD HH24:MI:SS') AND to_timestamp(:end_date, 'YYYY-MM-DD HH24:MI:SS')
```

Comment:
have we tested that the issue model still works with this change?

Reply:
This shouldn't affect any of the workers, but @ccarterlandis I am wondering why we made this change? I am assuming to make user inputs of begin and end dates more flexible?

Reply:
Yes, without adding the timestamp the dates were not being parsed correctly and the data was not being returned even it was there. However there have also been problems getting the data to gather correctly, which might have actually been the case